src/transformStateWithClones.js

```diff
@@ -6,8 +6,37 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
   // write code here
+  const array = [];
+  let stateCopy = { ...state };
+  let pushedCopy = {};
+
+  for (const object of actions) {
```

Comment:
I suggest to change the name of this constant
```suggestion
  for (const action of actions) {
```